cms/templates/asset_index.html

```diff
@@ -70,10 +70,11 @@ <h3 class="title-3">${_("Adding Files for Your Course")}</h3>
             <div class="bit">
                 <h3 class="title-3">${_("Using File URLs")}</h3>
 
-                <p>${_("Use the {em_start}Embed URL{em_end} value to link to the file or image from a component, a course update, or a course handout.").format(em_start='<strong>', em_end="</strong>")}</p>
+                <p>${_("Use the {em_start}Studio URL{em_end} value to link to the file or image from a component, a course update, or a course handout.").format(em_start='<strong>', em_end="</strong>")}</p>
```

Comment:
You should check with @singingwolfboy how to do this, as all references to "Studio" have been moved into theming settings.